src/components/RequestQueue.js

```diff
@@ -12,7 +12,7 @@ class RequestQueue extends Component {
   componentDidMount() {
     let requests = [];
     const db = firebase.firestore();
-    db.collection('requests').get()
+    db.collection('requests').where('active', '==', true).get()
```

Comment:
If you wanted to be really rigorous with your quality, you would probably want to have some file that defines the schema of your table and put constant strings like "active" in there. then you'd write:
```
db.collection(REQUESTS_TABLE_NAME).where(REQUESTS_TABLE.ACTIVE, '==', true).get()
```
Or however you decide to organize your constants. Though this may be too much for now, so it's a non-blocker.